language/Turkish/strings.po

```diff
@@ -5876,31 +5899,31 @@ msgstr ""
 
 #: plugins/cheatfind/init.lua:718
 msgid "Value"
-msgstr ""
+msgstr "Değer"
 
 #: plugins/cheatfind/init.lua:720
 msgid "Difference"
-msgstr ""
+msgstr "Fark"
 
 #: plugins/cheatfind/init.lua:725
 msgid "Any"
-msgstr ""
+msgstr "Herhangi"
 
 #: plugins/cheatfind/init.lua:731
 msgid "Data Format"
-msgstr ""
+msgstr "Veri Formatı"
 
 #: plugins/cheatfind/init.lua:737
 msgid "Slot 1 Value"
-msgstr ""
+msgstr "Slot 1 ,değeri"
 
 #: plugins/cheatfind/init.lua:737
 msgid "Last Slot Value"
-msgstr ""
+msgstr "Son Slot Değeri"
```

Comment:
The word “Slot” is used inappropriately in the English UI for the cheat finder plugin, and I don’t want this carried over to translations unnecessarily.

The way the cheat finder works is that it saves the contents of memory, and then scans for values that changed or didn’t change between the saved memory snapshots.  Right now, it refers to these saved snapshots of memory as “slots” in English.  This is a poor choice of terminology, since it doesn’t describe what they are, and it can cause confusion with what MAME calls slot devices.

Can you use a better term than “slot” in Turkish to refer to these snapshots of memory?

Reply:
fixed, last time i used cheat finder was 2004 when i found the cheats for "Twin Eagle" for Mame 0.81 xD